backend/src/config/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const QA = 'test';
+export const PROD = 'production';
+export const LOCAL = 'local';
```

Comment:
Maybe it will be better to organize in collection:
```suggestion
export const ENVIRONMENTS = {
  qa: 'test',
  prod: 'production',
  local: 'local',
};
```